addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -2181,6 +2182,64 @@ QUnit.test('chat window: chat header should not be clickable when thread has mul
     );
 });
 
+QUnit.test('2 chat windows: check shift operations have correct title with a LTR locale', async function (assert) {
```

Comment:
Test name is too obscure, doesn't clearly tell what is actually asserted in few words.
Test is about textual representation of "previous"/"next" being mapped appropriately to "left"/"right" in LTR locale.